### DISCUSSION TITLE
🐮 Animals [PROPOSAL]

### DISCUSSION THREAD

**Discussion by adamhannigan** [sunflower-land/sunflower-land#4300] [Ideas]
# 🐮 Animals

Yeeeeeehawww, animals are coming to Sunflower Land! The goal of animals is to provide new progression loops + engagement for new and long term players.

Players will be able to buy animals, keep them fed, collect resources & finally sell them at the exchange.

_This proposal is separate to the Pet Mechanic._

<img width="1482" alt="Screenshot 2024-08-27 at 9 22 05 AM" src="https://github.com/user-attachments/assets/cfa95004-7fdf-46da-ba7c-5a0c8d80cb5e">

_Art team sneak peek into animals!_

**Quick Note**

The costs and yields are mainly placeholders in this document. We calculate these based on progression times & engagement loops. For the ease of understanding, we have put some base numbers in this document to help convey the concept. 

Drop rates, buffs, requirements etc. will all be refined before launch.

### Buildings

Animals will no longer live outside the buildings. Players will need to craft new buildings to house the animals.

<table><thead><tr><th width="136">Name</th><th width="147">Animals</th><th width="97">Level</th><th>Costs</th><th>Animal Limit</th></tr></thead><tbody><tr><td>Hen House</td><td>Chickens</td><td>6</td><td>30 Wood, 5 Iron, 5 Gold</td><td>10</td></tr><tr><td>Barn</td><td>Cows, Sheep</td><td>14</td><td>150 Wood, 10 Iron, 10 Gold</td><td>10</td></tr></tbody></table>

_You can only have 1 Hen House (see below)_

When a building has resources to collect or animals to be fed, icons will appear above (similar to the Green House).

### Animals

Inside of the building, players can purchase new animals. When an animal is purchased, it appears on the floor.

<table><thead><tr><th width="146">Name</th><th width="103">Level</th><th>Cost</th></tr></thead><tbody><tr><td>Chicken</td><td>6</td><td>50 Coins</td></tr><tr><td>Cows</td><td>14</td><td>100 Coins</td></tr><tr><td>Sheep</td><td>18</td><td>120 Coins</td></tr></tbody></table>

### Crops

This season will introduce 1 new crop - Barley!

| Cost    | Sell Price | Lvl Required | Time     |
| ------- | ---------- | ------------ | -------- |
| 10 Coins | 12 Coins    | 14           | 48 hours |

_Barley is intended for animal feed and won't be used to earn coins (there are more profitable crops)_

There are no planned recipes for Barley.

### Feeder

Inside of the building, players will find a Feeder machine. Here you can turn resources into Feed for the animals. The feed is created instantly. When an animal is requesting a feed, you can click on the animal to apply it.

| Name         | Cost                      | Points |
| ------------ | ------------------------- | ---------------- |
| Kernel Blend          | 1 Wheat                   | 10               |
| Hay | 1 Corn                    | 15               |
| NutriBarley  | 1 Barley                  | 20               |
| Mixed Grain  | 1 Wheat, 1 Corn, 1 Barley | 35               |

**Bonus Items**

The following items can also be crafted and fed to animals for unique effects at any time (do not need to wait for animal to be hungry)

| Name          | Cost                | Utility                                                         |
| ------------- | ------------------- | --------------------------------------------------------------- |
| Fish Oil      | 5 Anchovy, 100 Kale | Instant Level Up (no resource drop)                                            |
| 20 x OmniFeed | 1 Block Buck        | Can be used instead of Hay/Kernel Blend/NutriBarley/Mixed Grain |
| Golden Glow   | 1 Gold              | Turns an animal "Gold" (requested at exchange)                  |
| Fire X        | 1 Crimstone         | Turns an animal "Red"                                           |
| Barn Delight  | 10 Eggs, 1 Iron     | Cures Sickness                                                  |
| Blue X        | 50 Blueberries      | Turns and animal "Blue"                                         |

### Affection Tools

| Name         | Cost        | Utility                       |
| ------------ | ----------- | ----------------------------- |
| Petting Hand | Free        | +2 Points                     |
| Brush        | 2000 Coins  | One time purchase (+3 points) |
| Music Box    | 50000 Coins | One time purchase (+5 points) |

### Keep em Happy!

<img width="705" alt="Screenshot 2024-08-27 at 3 23 38 PM" src="https://github.com/user-attachments/assets/44b62060-86f9-465d-a37e-5d6770a4ddde">

The goal of animals is to keep them happy. When an animal's happiness metre is filled, it will "Level up" and drop resources.

Every 24 hours an animal will ask for. When you apply the feed, the happiness metre will increase. _A feed does not guarantee an animal will level up._

This speed of feed request will be determined by any animal speed boosts you have.

Every **12 hours** an animal will also request some affection. Players can use the Petting Hand (Free), Brush, Music Box & Bucket to increase the animals happiness by X points.

_If you do not have the affection tool, you can wait until the animal requests feed again._

<table><thead><tr><th width="124">Animal</th><th width="76">Level</th><th width="158">Points to next lvl</th><th width="166">Requests</th><th>Lvl up Drops</th></tr></thead><tbody><tr><td>Chicken</td><td>1</td><td>15</td><td>Hay, Petting Hand</td><td>+1 Egg</td></tr><tr><td>Chicken</td><td>2</td><td>20</td><td>Hay, Petting hand</td><td>+2 Egg</td></tr><tr><td>Chicken</td><td>3</td><td>25</td><td>Hay, Brush</td><td>+2 Egg +1 Feather</td></tr><tr><td>Chicken</td><td>4</td><td>30</td><td>Kernel Blend, Brush</td><td>+2 Egg +2 Feather</td></tr><tr><td>Chicken</td><td>5</td><td>35</td><td>NutriBarley, Music Box</td><td>+3 Egg +2 Feather</td></tr><tr><td>Chicken</td><td>6</td><td>40</td><td>NutriBarley, Music Box</td><td>+3 Egg +3 Feather</td></tr><tr><td>Chicken</td><td>7+</td><td>45</td><td>NutriBarley, Music Box</td><td>+3 Egg +3 Feather<br>(Chance of getting mutants)</td></tr><tr><td></td><td></td><td></td><td></td><td></td></tr><tr><td>Sheep</td><td>1</td><td>15</td><td>Hay</td><td>+1 Wool</td></tr><tr><td>Sheep</td><td>2</td><td>20</td><td>Kernel Blend</td><td>+2 Wool</td></tr><tr><td>Sheep</td><td>3</td><td>25</td><td>Hay</td><td>+2 Wool +1 Merino Wool</td></tr><tr><td>Sheep</td><td>4</td><td>30</td><td>NutriBarley</td><td>+2 Wool +2 Merino Wool</td></tr><tr><td>Sheep</td><td>5</td><td>35</td><td>Mixed Grain</td><td>+2 Wool +2 Merino Wool</td></tr><tr><td>Sheep</td><td>6</td><td>40</td><td>Mixed Grain</td><td>+3 Wool +3 Merino Wool</td></tr><tr><td>Sheep</td><td>7+</td><td>45</td><td>Mixed Grain</td><td>+3 Wool +3 Merino Wool<br>(Chance of getting mutants)</td></tr><tr><td></td><td></td><td></td><td></td><td></td></tr><tr><td>Cows</td><td>1</td><td>15</td><td>Hay</td><td>+1 Milk</td></tr><tr><td>Cows</td><td>2</td><td>20</td><td>Hay</td><td>+2 Milk</td></tr><tr><td>Cows</td><td>3</td><td>25</td><td>NutriBarley</td><td>+2 Milk +1 Leather</td></tr><tr><td>Cows</td><td>4</td><td>30</td><td>NutriBarley</td><td>+2 Milk +2 Leather</td></tr><tr><td>Cows</td><td>5</td><td>35</td><td>Mixed Grain</td><td>+3 Milk +2 Leather</td></tr><tr><td>Cows</td><td>6</td><td>40</td><td>Mixed Grain</td><td>+3 Milk +3 Leather</td></tr><tr><td>Cows</td><td>7+</td><td>45</td><td>Mixed Grain</td><td>+3 Milk +3 Leather (Chance of getting mutants)</td></tr></tbody></table>

_At level 7+ players will begin to get the chance of mutants dropping_

_Can you level up a pet without feeding it? Yes, however this will take a very long time and is not great strategy for the Livestock exchange + sickness._

### Livestock Exchange

Players can open the Livestock each day to sell their animals. Each week **different** prices and offers are given - similar to a real exchange. This gives the player a **risky** decision whether or not to sell their animal. When prices are low, they would prefer to hold onto their animals - however, it also requires difficult feeds to sustain high level animals.

Each week there will be **5 one-time offers for animals for Seasonal Tickets**. Every player will get the same offer for fairness.

The animal level will be achievable within 5-6 days. This ensures players focussing on the season will be able to complete the challenge in time. A player can also use Fish Oil for instant levelling up - **Note, you do not receive resources when using Fish oil.**

There will also be **3 one-time offers for animals for SFL.**

Finally, a player will receive coin offers for the animals. They can sell **as many** animals that they would like at this price.

_Example offers:_

| Request                  | Offer      |
| ------------------------ | ---------- |
| Chicken (Lvl 5)          | 15 Tickets |
| Chicken (Lvl 4)          | 13 Tickets |
| Cow (Lvl 5)              | 16 Tickets |
| Sheep (Lvl 2)            | 6 Tickets  |
| **Gold** Chicken (Lvl 3) | 400 Coins  |
| Cow (Lvl 2)              | 200 Coins  |
| ....                     |            |

If a player accepts an offer, they then pick which animal they want to sell.

### Sickness

The older an animal is, the more likely it will get sick. This introduces some loss aversion into the game where players risk keeping an animal alive for too long (instead of selling it).

When an animal becomes sick, it will drop half the amount of resources & sell for 25% less at the exchange.

A player has 2 options:

1. Sell the animal at the exchange (infected animals receive 25% less coins)
2. Craft the Barn Delight an cure the animal

Every 24 hours, an animal has the chance of becoming sick. This is based on how many days old they are (since they were bought).

| Days old | Chance |
| -------- | ------ |
| 1        | 0%     |
| 2        | 0%     |
| 3        | 1.5%   |
| 4        | 2.5%   |
| 5        | 3.5%   |
| 6        | 4%     |
| 7        | 5%     |
| 8        | 6%     |
| 9        | 7%     |
| 10+      | 10%    |

Note, sickness will spread! For each sick animal in the building, there is an additional 2.5% chance an animal will get sick.

### Onboarding

To get new players engaged, we want to reduce the idle time when the buildings are crafted. We don't want them waiting 24 hours until they can understand the full loop of feeding animals + collecting resources.

When a Hen House or Barn is built, it will come with 3 Chickens or 3 Cows respectively.  A player will also be given 5 free feed. Each of these animals will already have 5 points, which means they will instantly level up when the feed is given to them. A player should be able to attain Eggs + Milk instantly on their **first time**.

This will help them grasp how the mechanic works.

### Animal Resource Utility

An animal drops produce and rare crafting materials.

| Name        | Utility             |
| ----------- | ------------------- |
| Eggs        | Food                |
| Milk        | Food                |
| Leather     | Crafting |
| Feathers    | Crafting |
| Wool        | Crafting |
| Merino Wool | Crafting |

#### Crafting Box

Introduce a new building in a similar style to Minecraft crafting. Players combine resources to produce items.\
\
These items can be requested in deliveries by each NPC, or used as cosmetics on their farm/bumpkin.

| Name                | Requires                 | Time       |
| ------------------- | ------------------------ | ---------- |
| Path                | 1 Stone                  | Instant    |
| Fence               | 3 Wood                   | Instant    |
| Stone Wall          | 3 Stone                  | Instant    |
| Toad Seat           | 3 Mushroom, 2 Wood       | 5 minutes  |
| White Tulips        | White Pansy, Stone       | 5 minutes  |
| Potted Sunflower    | Sunflower, Stone         | 5 minutes  |
| Potted Potato       | Potato, Stone            | 5 minutes  |
| Potted Pumpkin      | Pumpkin, Stone           | 5 minutes  |
| Basic Bear          | 7 Wool                   | 10 minutes |
| Bonnies Tombstone   | 5 Stone                  | 15 minutes |
| Grubnash Tombstone  | 5 Iron                   | 15 minutes |
| Farm Sign           | 5 Wood                   | 15 minutes |
| Basic Wig           | 5 Feather                | 15 minutes |
| Rancher Hair        | 1 Basic Wig + 5 Carrots  | 15 minutes |
| Basic Shirt         | 5 Wool                   | 15 minutes |
| Red Farmer Shirt    | 5 Wool + 5 Radish        | 15 minutes |
| Farmer Pants        | 7 Leather                | 15 minutes |
| Farmer Overalls     | Farmer Pants + 2 Leather | 15 minutes |
| Lumberjack Overalls | Farmer Pants + 5 Wood    | 15 minutes |

<img width="583" alt="Screenshot 2024-08-27 at 3 25 42 PM" src="https://github.com/user-attachments/assets/a4ba991a-4f8f-4bd6-b078-4a2c98c28a67">

#### Megastore Clothing

All Megastore clothing will now require animal materials to craft. (Leather, Wool, Feathers, Merino Wool)

**Bumpkin Beds**

Craft beds for more Bumpkins on farm. This will replace Block Bucks for additional Bumpkins.

| Bumpkin # | Cost                                              |
| --------- | ------------------------------------------------- |
| 1         | Free                                              |
| 2         | 50 Wood 5 Feathers                                |
| 3         | 50 Wood 25 Features 15 Leather 10 Wool            |
| 4         | 50 Wood 50 Features 50 Leather 50 Wool            |
| 5         | 50 Wood 100 Features 80 Leather 70 Wool           |
| 6         | 50 Wood 150 Features 120 Leather 50 Merino Wool  |
| 7         | 50 Wood 200 Features 150 Leather 75 Merino Wool  |
| 8         | 50 Wood 250 Features 200 Leather 150 Merino Wool |

_There is a small handful of players with leftover **farm hand coupons**. These will be compensated with 15 BBs per coupon._

### Upgrades

A player will be able to upgrade the buildings to unlock more spots for animals:

| Building    | Upgrade    | Cost        |
| ----------- | ---------- | ----------- |
| Hen House 2 | +5 Animals | 7500 Coins + Materials  |
| Hen House 3 | +5 Animals | 50000 Coins + Materials |
| Barn 2      | +5 Animals | 10000 Coins + Materials |
| Barn 3      | +5 Animals | 75000 Coins + Materials |

_Players with an additional Hen House will be able to sell one at the Garbo - the option to sell should only appear if they have 2 Hen Houses. Players will be able to sell for the same (likely a bit more) value in coins that they crafted their second Hen House for_

**Equipment**

A player will also be able to unlock the following Equipment at the Workbench

| Name | Cost                          | Utility   |
| ---- | ----------------------------- | --------- |
| Bale | 200 Eggs, 50 Wood, 1000 Coins | +0.2 Eggs |
| ?    | 25 Milk, 50 Wood, 1000 Coins  | +0.1 Milk |
| ?    | 25 Wool, 50 Wood, 1000 Coins  | +0.1 Wool |

### Landscaping

Players will be able to organise inside of their animal buildings. This will help them group animals by type and/or level.

They can also place their mutant Chickens + animal related equipment in here.

### NFTs

With the new animal mechanic, certain in-game items will be adapted to the new system

<table><thead><tr><th width="223">Name</th><th>Utility</th></tr></thead><tbody><tr><td>Gold Egg (Cockerel)</td><td>Feed chicken without a feed (e.g Hay, Kernel Blend, Omnifeed etc.)<br>Does not include bonus items/tools</td></tr><tr><td>Chicken Coop</td><td>+1 Eggs, Mega Hen House (Room for 30 chickens) - no need to build or upgrade Hen Houses</td></tr><tr><td>CattleGrim</td><td>Produce yield - +0.25 Eggs, +0.25 Milk, +0.25 Wool</td></tr><tr><td>Retreat Buds</td><td>Produce yield - +0.2 Eggs, +0.2 Milk, +0.2 Wool</td></tr><tr><td>Mutant Chickens</td><td>Same egg boosts + multipliers</td></tr><tr><td>Rooster</td><td>Same mutant boost (when 7+)</td></tr><tr><td>Farm Dog</td><td>+25% Sheep Speed</td></tr></tbody></table>

Legacy animal skills will also be supported in this system.

**Upcoming NFTs**

This proposal excludes the upcoming NFTs. These will be launched and proposed with the upcoming seasonal launch document.

### Recipes

We will include Milk recipes

**Comment by Blasta-alasta** [comment]
Wow looks amazing....Don't forget Retreat Buds!

**Reply by adamhannigan** [reply]
Ah miss, I forgot them in this proposal. It is their time to shine!

**Comment by brian-man** [comment]
I love seeing these proposals to get community feedback early in the process.

First pass notes:
1. 404 for "upgrades" link in Buildings section
2. Can bumpkins eat barley (and cook with it)?
3. Numbers aside, what's the benefit of Mixed Grain? (Why does it need to exist?)
4. Affection Tools: Is the utility "Points" the "Affection Points" (different from "Happiness Points")?
5. For the change in Chicken, it seems like the egg production rate of the community as a whole will be a lot lower than now.  Will need some retuning of all existing Egg demand.
6. "do not receive resources when using Fish oil" - This subtlety will trip up and annoy players a lot.  It will add noise to discord for sure.
7. Sickness spreading sounds awful.  Too RL for my taste.  Especially with recent pandemic.
8. Garbo price for selling second Hen House needs to be retuned.  The current cost for crafting is      
800 coins, 200 Wood, 15 Iron, 15 Gold, and 300 Egg.  At current P2P market prices, that's approximately 21000 coins.  So the Garbo price of 7500 coins is quite a bit off the mark.
9. "Golden Egg" is actually "Gold Egg".

**Reply by adamhannigan** [reply]
Some great points. I have updated the doc to address a lot of these 👍 

`Numbers aside, what's the benefit of Mixed Grain? (Why does it need to exist?)`

Mixed Grain stands as an expensive feed which attempts to constrain how many levels an animal will reach - i.e. it becomes very expensive to maintain at high levels.

`For the change in Chicken, it seems like the egg production rate of the community as a whole will be a lot lower than now.`

Yes, we will likely update Egg requirements in baking + compositing + deliveries to match the new expected supply/demand.

**Comment by snorri-snorri** [comment]
I know a new player just bought his first chicken, 200 coins, could be upset if by the time he gets maximum the price drops to 50 coins.

I feel if there is a price change to come, chickens could be best changing ASAP to prevent people getting annoyed

**Comment by ispankzombiez** [comment]
I think an egg incubator would be a really nice additional building as well :) being that chickens are sellable with this proposal, it would be really nice to be able to incubate a few eggs at a time. (Ofc this would be very slow, not many eggs at a time)

**Comment by RicardoKeita** [comment]
Every 24h they will ask for feed? What will happen with old NFTs and legacy skills that boost speed (chikens)? I feel like it's throwing those away...

Chiken coop, say it doubles...why change it? If you going to increase the amount on the base reward I sugest to not go with this absurd double and triple amounts...It is just too high that needs to nerf the old nft to balance. Why not balance the future to fit?

Sickness with days old are increasing in a pace really fast, this feels the loss will happens anyway, and we will have to manage the huge or lower loss.

**Reply by raulmonoanivia** [reply]
1st - As stated on the proposal, the boosts will reduce the time to be fed based on the new 24h timer
![image](https://github.com/user-attachments/assets/010e426d-a1f7-4ffb-9e3e-a9a390a67aa1)

2nd - 
![image](https://github.com/user-attachments/assets/3b81f5de-43a9-484c-99de-efac411b1feb)
Chicken Coop is giving the exact same boost as it gives today, with exception of the new feature of not having to craft any hen house

3rd - If you're that worried about sickness, just sell them asap when they get to 10 days old. And if they get sick before, sell them for the -25% coin value to prevent the healthy ones of the increased sick chance, i'm sure it will still be a profit

**Reply by RicardoKeita** [reply]
It's +1 because it doubles the base amount, that it is +1. If the chiken it's rewarding the base amount +2, it's another history, that would be chicken coop increasing +2 (in the actual read about how those buffs works defended by devs). I don't really want to chicken coop be this absurd, just want to say to look at that more carefully...if they really want to maitain it's just say the lvl up chiken would reward buffs like +1, +2 and the base amount still +1 <- what seems to be (but make it clear). 
But this means a lot of eggs will flow into market and I remember i have heard about egg inflation at some period, i don't have the numbers to say what's the best for everyone, but maybe they want a lot of more eggs on market...that's another history. More eggs, means more need of burn or the eggs will lose value.

Yeah, I am really worried about sickness RNG since I don't like this gamble...

**Reply by raulmonoanivia** [reply]
It's not doubling the base amount, you can check the boost tab in sfl.world and the github documents. It's a fixed +1 Egg per chicken

**Reply by RicardoKeita** [reply]
You are right, my mistake.

**Comment by Xeko-92** [comment]
I'm curious about this part:

>Every 12 hours an animal will also request some affection

Does that mean that you have a 12 hour window to do the affection? Or will the timer start once you do the affection action? I hope it's the former, similar to the fishing times where they're set times during the day rather than only start once you do the action, requiring you to be on immediately once the time ends.

**Reply by adamhannigan** [reply]
This is still open for decision 👍 I see some pros and cons in both approaches.

Ideally since it is every 12 hours we want to ensure these are the periods when a player starts in the morning (i.e. they wake up) and before they go to sleep

**Reply by Xeko-92** [reply]
I personally think it should follow the same as fishing times, dawnlight and dusktide. We don't want to get into a place where people aren't able to keep up, especially with all the new stuff being added. Minigames, digging. It would be a lot if you also had to track when you last played with your pet to make sure you're on in exactly 12 hours.

**Comment by raulmonoanivia** [comment]
First of all, I'm gonna say that this proposal is so good that it made me want to cry tears of joy.
Second of all, i noticed these typos that should be corrected for more clarity xD
![image](https://github.com/user-attachments/assets/4c63c0df-5eef-4894-8d47-dc4a3cfbe5f6)
![image](https://github.com/user-attachments/assets/72994488-ae67-4fc2-af98-22f77dc3907d)

Regarding the proposal itself, I loved most of it, but here are the points I think could be reviewed:
• Selling the second hen house for 7k coins is way too low and it will make a lot of people (that already owns them) angry, my personal belief is that it should give enough coins to cover the costs of the Upgrade to lv2 + Half of the upgrade to level 3 (so 32,5k coins)
• Forgot to include bud animal boosts in the NFT section
• If the animal is asking for a specific feed, could we still give a "better one" to them? Or just precisely the one that they are asking?

And for now I think that's it :)
Great job with the whole crafting, sickness, upgrades and the whole rest, i'm really excited for this to arrive

**Reply by adamhannigan** [reply]
Thanks, I've made some updates to reflect this 👍 

The "better one" feed is an interesting idea. It would definitely if you've over supplied on a certain crop

**Comment by heatedlime** [comment]
Wow this is awesome. Tbh, this sounds way better than factions and digging. Great work.

**Comment by raulmonoanivia** [comment]
![image](https://github.com/user-attachments/assets/155eb868-38ec-45ff-9452-a88c4d979c09)
Also one more thing cause i just noticed it. The boost description says "animal" instead of "chicken" Does this mean that Gold Egg will provide free food for every animal? Or just the chickens as currently is?

**Reply by adamhannigan** [reply]
Thanks, that was a typo. It should be "chicken". Fixed ✅

**Comment by fra-vh** [comment]
sickness seems too harsh. I suggest:

-make % chance of getting sick linked to lvl of the animal (easier to keep track of)
-no spreading sickness, too harsh for no good reason
-make sick pets drop 50% less of everything  (to compensate for the other changes)

**Reply by adamhannigan** [reply]
We will do some beta testing around the sickness levels and ensure it isn't too disruptive 👍 

I really like the idea of sick animals dropping less though 😈 I've added it to the proposal! Thanks

**Reply by RicardoKeita** [reply]
I think spreading sickness is something nice for maintaining some kind of reality risk, but it should be a balacing reward at the risk return. Also, i would like to see some kind of choose to make about maintaning old chickens or not.

**Comment by brian-man** [comment]
Undead Rooster is missing from your list.

**Reply by adamhannigan** [reply]
Good spot 👍

**Comment by ShinKan42** [comment]
should really consider at how much the animals can be sold
-25% price can be nothing when compared to the mutant chances
so one of the strategy and which is not desired from this mech so it seems is that keeping all animals affected to max out the mutant chance

**Reply by adamhannigan** [reply]
Definitely. We want it to be a difficult decision to make - I do think overall it will make mutants a bit more challenging and risky to go for

**Reply by elowine** [reply]
> Définitivement. Nous voulons que ce soit une décision difficile à prendre - je pense que dans l'ensemble, cela rendra les mutants un peu plus difficiles et risqués

if the aim is to have an animal replacement mechanism, I think we need to reduce the chance of mutants and the number of eggs obtained after a certain time.

In real life, for example, a hen produces regularly, and when she's 4/5 years old, she doesn't produce every day.

**Reply by RicardoKeita** [reply]
> if the aim is to have an animal replacement mechanism, I think we need to reduce the chance of mutants and the number of eggs obtained after a certain time.
> 
> In real life, for example, a hen produces regularly, and when she's 4/5 years old, she doesn't produce every day.

Real nice approach, I vote for something like this.
Maybe doing the number of eggs buff be a chance increasing for more with higher lvls...

<html>
<body>

Animal | Level | Points to next lvl | Requests | Lvl up Drops


(One week chickens)
Chicken | 1 | 15 | Hay, Petting Hand | +1 Egg
Chicken | 2 | 20 | Hay, Petting hand | +1 egg + (50% chance of +1 Egg)
Chicken | 3 | 25 | Hay, Brush | +1 Egg + (75% chance of +1 Egg) + 1 Feather
Chicken | 4 | 30 | Kernel Blend, Brush | +2 Egg +2 Feather
Chicken | 5 | 35 | NutriBarley, Music Box | +2 Egg + (50% chance of +1 Egg) +2 Feather
Chicken | 6 | 40 | NutriBarley, Music Box | +2 Egg + (75% chance of +1 Egg) +2 Feather
Chicken | 7 | 45 | NutriBarley, Music Box | +3 Egg + (50% chance of +1 Egg)  +3 Feather (Chance of getting mutants)

(Two week chickens = increasing risk, "mutants or eggs")
Chicken | 8 | ? | ? |  +3 Egg + (50% chance of +1 Egg)  +3 Feather (Chance of getting mutants and sickness, +0.25 %)
Chicken | 9 | ? | ? |  +3 Egg  +3 Feather (Chance of getting mutants and sickness, +0.5 %)
Chicken | 10 | ? | ? |  +2 Egg + (75% chance of +1 Egg)  +3 Feather (Chance of getting mutants and sickness, +0.75 %)
Chicken | 11 | ? | ? |  +2 Egg + (50% chance of +1 Egg)  +3 Feather (Chance of getting mutants and sickness, +1 %)
Chicken | 12 | ? | ? | +2 Egg +3 Feather (Chance of getting mutants and sickness, +1.25 %)
Chicken | 13 | ? | ? | +1 Egg + (75% chance of +1 Egg)  +3 Feather (Chance of getting mutants and sickness, +1.5 %)
Chicken | 14 | ? | ? | +1 Egg + (50% chance of +1 Egg)  +3 Feather (Chance of getting mutants and sickness, +1.75 %)

(Old Chikens = righer risk, "mutants or eggs")
Chicken | 14+ | ? | ? | +1 Egg +1 Feather (Chance of getting mutants and sickness, +2 %) (maybe here can add a option for burning for food "grilled chiken" buff +100% XP)


</body>
</html>

Not balanced, just an idea...

**Reply by adamhannigan** [reply]
Great ideas!

We are going to experiment with a few of these approaches over the coming weeks 🤗

**Comment by heatedlime** [comment]
Would be cool to give the farm cat a use / boost. Current description is “Keep the rats away with this rare item”. 

What if rats at your farm where what carry diseases, which cause your animals to become sick, and the farm cat decreases the chance of your animals becoming sick?

**Reply by heatedlime** [reply]
Also, you could create a new craft-able item/boost. A mouse trap. Could cost 1-2 wood and would decrease the chance (5-10%?) your animals get sick for 6ish hours every. Great resource burn, and makes people play the game more often to place a mouse trap in their barn.

**Comment by LucasAG95** [comment]
"I don't know if this is already planned, but I'd like to suggest an idea for using milk in cakes. I believe it won't change their recipes, and it's actually better if it doesn't. What I think could be done is to treat milk like 'Oil' - by adding an amount to the Bakery, it makes the cake 'creamier' and thus gives more XP. How it would be applied would be up to you guys, it could really be like Oil, where an amount affects so many cakes according to the XP it gives, or when cooking you could select whether to add milk to the cake or not. I'm not sure if I expressed the idea well, but that's it."

**Comment by Xeko-92** [comment]
So Andrei posted the information for Chicken levels and the XP requirement is quite different from what was posted here. I'm not sure what the point of posting X and then putting Y into the main game is if what you're looking for is feedback from us. Instead of 45 xp per level at 7+, we need more and more experience at every level.
